src/main/java/com/baidu/hugegraph/api/traverser/RingsAPI.java

```diff
@@ -31,4 +43,39 @@ public RingsAPI(RestClient client, String graph) {
     protected String type() {
         return "rings";
     }
+
+    public List<Path> get(Object sourceId, Direction direction, String label,
+                          int depth, boolean sourceInRing, long degree,
+                          long capacity, long limit) {
+        String source = GraphAPI.formatVertexId(sourceId, false);
+
+        checkPositive(depth, "Max depth of path");
+        checkDegree(degree);
+        checkCapacity(capacity);
+        checkLimit(limit);
+
+        Map<String, Object> params = new LinkedHashMap<>();
+        params.put("source", source);
+        params.put("direction", direction);
+        params.put("label", label);
+        params.put("max_depth", depth);
+        if (sourceInRing) {
+            Version apiVersion = this.client.apiVersion();
+            if (apiVersion != null &&
```

Comment:
move check of `apiVersion != null` to client.apiVersion()

Reply:
apiVersoin is null if use RestClient directly instead of using HugeClient.